Autopilot/Src/tim.c

```diff
@@ -56,9 +56,33 @@
 
 /* USER CODE END 0 */
 
+TIM_HandleTypeDef htim6;
 TIM_HandleTypeDef htim10;
 TIM_HandleTypeDef htim11;
 
+/* TIM6 init function */
+void MX_TIM6_Init(void)
+{
+  TIM_MasterConfigTypeDef sMasterConfig;
+
+  htim6.Instance = TIM6;
+  htim6.Init.Prescaler = 108;
+  htim6.Init.CounterMode = TIM_COUNTERMODE_UP;
+  htim6.Init.Period = 65000;
```

Comment:
Pretty sure these Prescaler and Period settings are wrong. Clock division is `FCLK / (PSC + 1)`, and I don't see a reason to use 65000 for the period instead of 65535 (0xFFFF).